schema.yml

```diff
@@ -0,0 +1,504 @@
+openapi: 3.0.3
+info:
```

Comment:
why do you commit schema?

Reply:
I don't need to since it's generated automatically and for my purposes I don't need this file :)